lib/c.c

```diff
@@ -436,80 +440,89 @@ int fputc(int c, FILE *stream)
     return 0;
 }
 
-/* Reference:
- *   https://danluu.com/malloc-tutorial/
- * FIXME: adopt lite_malloc from musl-libc
- *   https://git.musl-libc.org/cgit/musl/tree/src/malloc/lite_malloc.c
- */
+#define PROT_READ 0x01
```

Comment:
Don't define these. Instead, specify pre-calculated values as the parameters in system calls.